packages/recordio/README.md

```diff
@@ -0,0 +1,39 @@
+# RecordIO
+
+Package exports one function that parses RecordIO encoded records.
+
+## Usage
+
+```javascript
+import read from "recordio";
+
+const [records, rest] = read(input);
+```
+
+Function `read` returns a tuple whose first element is an array of records and the second element is the rest part of the input that is either empty or contains partial records.
+
+## RecordIO format
+
+This documentation can be also found in the [Apache Mesos documentation](http://mesos.apache.org/documentation/latest/scheduler-http-api/#recordio-response-format).
```

Comment:
Let's rephrase this paragraph instead of "copying" it from Mesos. I'd extract only the important bits and then link to the details Mesos documentation.